### PR TITLE
feat(react/webchat-emojipicker): emojiPicker added in user input box

### PR DESCRIPTION
**Pull request by imOscarCrespo**
I've add a new feature to have an emojipicker in our footer box. To call this menu it's necessary to declare in the index.js of our bot : " emojiPicker: true". I also refactored some styles to have a better responsive properties.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,5 +19,8 @@
   },
   "workspaces": [
     "packages/*"
-  ]
+  ],
+  "dependencies": {
```
Comment (dpinol) [inline package.json]: dependency should be added instead to react package

File: packages/botonic-react/src/webchat/webchat.jsx [hunk 2]
```diff
@@ -154,6 +155,9 @@ export const Webchat = forwardRef((props, ref) => {
   const handleMenu = () => {
     menuIsOpened ? setMenuIsOpened(false) : setMenuIsOpened(true)
   }
+  const handleEmoji = () => {
+    emojiIsOpened ? setemojiIsOpened(false) : setemojiIsOpened(true)
```
Comment (dpinol) [inline packages/botonic-react/src/webchat/webchat.jsx]: setemojiIsOpened => setEmojiIsOpened

where is setemojiIsOpened defined?

File: packages/botonic-react/src/webchat/webchat.jsx [hunk 3]
```diff
@@ -460,15 +462,32 @@ export const Webchat = forwardRef((props, ref) => {
...
                   border: 'none',
                   resize: 'none',
                   overflow: 'auto',
                   outline: 'none',
-                  marginTop: '13px'
+                  flex: '1 1 auto',
+                  padding: '14px 10px 0px 0px'
                 }}
               />
+              {props.emojiPicker && (
```
Comment (dpinol) [inline packages/botonic-react/src/webchat/webchat.jsx]: this method is too large.
Please create in this PR a method like renderEmojoPicker with this code, and in another PR try to split this huge return in smaller chunks. 
thanks

File: packages/botonic-react/src/webchat/webchat.jsx (hunk 2 above)
Comment (imOscarCrespo) [reply packages/botonic-react/src/webchat/webchat.jsx]: here : const [emojiIsOpened, setemojiIsOpened] = useState(false)
It's a react hook state.

File: packages/botonic-react/src/webchat/webchat.jsx (hunk 3 above)
Comment (imOscarCrespo) [reply packages/botonic-react/src/webchat/webchat.jsx]: what method? this is not a method. It's only a condition for seeing if user has declared emojiPicker in his index.js file

File: packages/botonic-react/src/webchat/webchat.jsx (hunk 2 above)
Comment (dpinol) [reply packages/botonic-react/src/webchat/webchat.jsx]: thanks, please rename it to setEmojiIsOpened

File: packages/botonic-react/src/webchat/webchat.jsx (hunk 3 above)
Comment (dpinol) [reply packages/botonic-react/src/webchat/webchat.jsx]: I meant that the function it belongs to is too long. Please Move lines 475 to 490 to a different method in the is PR. In another PR, please do the same to simplify further the Webchat return (from line 336 till the end of the file).
Ideally it should look st like:
``` return (
    <WebchatContext.Provider
      value={{
        sendText,
        sendPayload,
        openWebview,
        resolveCase,
        webchatState,
        addMessage,
        toggleWebchat,
        updateMessage,
        updateReplies,
        staticAssetsUrl
      }}
    >
   {renderTriggerButton(webChatState)}
   {webchatState.isWebchatOpen && (
        <div
          style=divStyle(webchatState)
        >
         header(toggleWebchat)
         messageList(webchatState)
         replies(webchatState)
         emojiIsOpened && emojiPicker()
         persistenMenu(...)
        ....
```

**Pull request merged**